app/Http/Controllers/Api/CtsController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Repositories\Cts\BookingRepository;
+use Illuminate\Http\Request;
+
+class CtsController
+{
+    private $bookingRepository;
+
+    public function __construct(BookingRepository $bookingRepository)
+    {
+        $this->bookingRepository = $bookingRepository;
+    }
+
+    public function getBookings(Request $request)
+    {
+        $bookings = $this->bookingRepository->getBookings(50);
+
+        if ($bookings->isEmpty()) {
+            return response()->json(['message' => 'No bookings found'], 404);
+        }
+
+        // Not all bookings from the min/max date may be returned if the pagination limit is reached
+        $fromDate = $bookings->first()->date;
+        $toDate = $bookings->last()->date;
+
+        return response()->json([
+            'bookings' => $bookings,
```

Comment:
I think we'd like to be quite selective with the fields we are surfacing via this API. If you can identify the bare minimum fields, I think that would be useful. For example: `member_id` isn't actually their CID... I can talk more about that via DM if it's useful.

Looking at the schema I'd suggest plucking the following fields:
- id
- date
- from
- to
- position
- type